server/shared/store/providers/local.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const isNil = require('lodash/isNil');
+const LRU = require('lru-cache');
+const micromatch = require('micromatch');
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  maxAge: yup.number()
+});
+
+class Local {
+  constructor(config) {
+    config = schema.validateSync(config, { stripUnknown: true });
+    this.name = 'local';
```

Comment:
What's the purpose of this field, I can't find it being used anywhere?

Reply:
The purpose is that consumer can get information about the key-value store provider which is being used. In addition, a consumer can make some actions depend on the store type. 
It is not used anywhere in this PR but I find it potentially useful for future store consummation. What do you think?

Reply:
@ikovac it's easy to add this if necessary, so I suggest we remove it to avoid piling up dead code.
Also, we could do this check with the `instanceof` operator and avoid having to change multiple things to preserve such behavior in the future. 😄